src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
+    protected String deadline;
+
+    public Deadline(String description, char taskType, String deadline) {
+        super(description, taskType);
+        this.deadline = deadline;
+    }
+
+    public String printTask() {
```

Comment:
slightly misleading method name as it suggests the use of System.out.print or an equivalent, but instead it returns a string